content/docs/command-reference/remote/modify.md

```diff
@@ -262,6 +262,22 @@ methods that are performed by DVC (`list_objects_v2` or `list_objects`,
   $ dvc remote modify --local myremote sse_kms_key_id 'key-alias'
   ```
 
+- `sse_customer_key` - user specified key to encrypt data uploaded when using
+  SSE-C. The value should be base64 encoded version of 256 bit key.
```

Comment:
Hi! What's the value provided to `sse`? Is it `SSE-C`, or something like `aws:ssec` (similar to `aws:kms`) ?

Reply:
Also, this is AWS-specific right?

Reply:
Yes. This is AWS-specific. Also, when SSE with client side keys for encryption is in question, libraries usually expect different parameter (`SSECustomerAlgorithm` as key in s3_additional_kwargs dictionary) from sse (which gets added as `ServerSideEncryption` as key in the s3_additional_kwargs dictionary). So, for SSE, when a user wants to use key in aws kms, the relevent parameters are `sse` and `sse_kms_key_id`. When a user wants to use their own key (which is SSE-C) then the relevant parameters are `sse_customer_key` and `sse_customer_algorithm`. Value of `sse` is irrelevant.

Reply:
Got it.